fonts.js

```diff
@@ -0,0 +1,42 @@
+// Synchronously load TTF fonts.
+// First, have Webpack load their data as Base 64 strings.
+/* eslint-disable global-require */
+const FONTS = {
+	// @todo get suggestions for font names and maybe replace these names
+	'Typey McTypeface': require('base64-loader!scratch-render-fonts/NotoSans-Medium.ttf'),
+	'Seriffy McSerifface': require('base64-loader!scratch-render-fonts/SourceSerifPro-Regular.otf'),
+	'Handlee McHandface': require('base64-loader!scratch-render-fonts/handlee-regular.ttf'),
+	'Knewey McKneeface': require('base64-loader!scratch-render-fonts/knewave.ttf'),
+	'Griffy McGriffface': require('base64-loader!scratch-render-fonts/Griffy-Regular.ttf'),
+	'Gameface': require('base64-loader!scratch-render-fonts/PressStart2P-Regular.ttf'),

```

Comment:
I assume LLK has permission to use these fonts and change the names they're presented as in the editor? (I'm also curious what the reasoning is behind renaming the fonts, rather than using their official names, although this was probably discussed offline.)

Reply:
Yes, another reason we're replacing all of our fonts is so that the new ones will all have open font licenses

Reply:
The names will almost definitely change :p

Reply:
"almost"